lib/colorado_lottery.rb

```diff
@@ -0,0 +1,24 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    if contestant.age <= 18
+      false
+    elsif contestant.age >= 18
+      contestant.game_interests.any? do |interest|
+         interest == game.name
+      end
+    else
+      false
+    end
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) && ((game.national_drawing?) || !contestant.out_of_state?)
```

Comment:
If `interested_and_18?` was set up properly then this is exactly the setup you needed for this to run properly. But, you are missing a closing parentheses at the end of the statement.